src/components/nvml/linux-nvml.h

```diff
@@ -16,6 +16,9 @@
 #define FEATURE_POWER_MANAGEMENT 1024
 #define FEATURE_NVML_POWER_MANAGEMENT_LIMIT_CONSTRAINT_MIN 2048
 #define FEATURE_NVML_POWER_MANAGEMENT_LIMIT_CONSTRAINT_MAX 4096
+#define FEATURE_ENERGY_CONSUMPTION 8192
```

Comment:
Change the name from `FEATURE_ENERGY_CONSUMPTION` to `FEATURE_TOTAL_ENERGY_CONSUMPTION`.